src/library/Core.nim

```diff
@@ -238,6 +238,17 @@ proc defineSymbols*() =
         attrs       = NoAttrs,
         returns     = {Any},
         example     = """
+            a: null
```

Comment:
I think there is a problem with the `coalesce` documentation.

Basically, it's nothing but the [nil-coalescing operator](https://www.hackingwithswift.com/example-code/language/what-is-the-nil-coalescing-operator)...

Let me explain...

Let's say we have this:

```
x: attr "myAttr"  ; gets attribute `.myAttr` if it doesn't exist it returns null
if? not? null? x -> print x
else -> print "attr not found"
```

The equivalent with `coalesce` would be:

```
print coalesce attr "myAttr" "attr not found"
```

or... with `??`:

```
print (attr "myAttr") ?? "attr not found"
```

😉